lib/src/modules/person/Person.js

```diff
@@ -31,7 +31,12 @@ export function Person({
   const separatedName = highlightText ? (
     <span>
       {name.slice(0, indexOfHighlightedText)}
-      <b>{highlightText}</b>
+      <b>
+        {name.slice(
+          indexOfHighlightedText,
+          indexOfHighlightedText + highlightText.length
+        )}
+      </b>
```

Comment:
We used to render `highlightText` here. However, the search is case insensitive so it would sometimes change the casing of the person's name. So now we will render the person's name instead!